From 44b193235b138c5cfb885795b49e00dbdbd12fae Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Mon, 23 May 2016 16:16:31 +0300
Subject: [PATCH 260/388] staging: fsl-dpaa2/mac: don't print DPMAC API version

Remove unnecessary message at probe, as we intend to support
more than one DPMAC API version.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index 366ad4c..e27a8d3 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -532,9 +532,6 @@ static int dpaa2_mac_probe(struct fsl_mc_device *mc_dev)
 		goto err_close;
 	}
 
-	dev_info_once(dev, "Using DPMAC API %d.%d\n",
-		      priv->attr.version.major, priv->attr.version.minor);
-
 	/* Look up the DPMAC node in the device-tree. */
 	dpmac_node = lookup_node(dev, priv->attr.id);
 	if (!dpmac_node) {
-- 
2.9.3

